sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/expressions/codegen/CodeFormatterSuite.scala

```diff
@@ -36,6 +36,22 @@ class CodeFormatterSuite extends SparkFunSuite {
     }
   }
 
+  test("removing overlapping comments") {
+    val code = new CodeAndComment(
+      """/*project_c4*/
+        |/*project_c3*/
+        |/*project_c2*/
+      """.stripMargin,
+      Map(
+        "/*project_c4*/" -> "// (((input[0, bigint, false] + 1) + 2) + 3))",
```

Comment:
Hi, @cloud-fan .
This is only targeting for those overlapping expressions.
It's not for multi-line comments.
Could you give me some kind of `generated overlapping multi-line comment`?
If then, I can grasp your concerns more fast.